### PR TITLE
[feature request] Optimize mobile layout How-to and Research #2735

### LINKED ISSUE #2735: [feature request] Optimize mobile layout How-to and Research
### Description
It's getting busy in the headers and it could use some optimisation on mobile. As mention in issue #2191
This issue is a follow up from https://github.com/ONEARMY/community-platform/issues/2734

We want to add hide some more detailed information by default that can be shown on a click using the "accordion" style

### Page related

Will be used in :
[Research](https://dev.onearmy.world/research/more-draft-checking)
[How-to](https://dev.onearmy.world/how-to/test-title)

### Mockup
Example for the research module, same principle applies to how-to's
afbeelding

### Note
It contains the new stats from #2734 but also moves the contributors on mobile.

### PR DESCRIPTION
**Pull request by denyilm**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

- adds FaChevronUp and FaChevronDown icons
- adds a more information component to the ContentStatistics component that becomes visibe in the mobile layout

## Git Issues

Closes #2735 

## Screenshots/Videos

image
image
image

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of our monthly maintainers call (first Monday of the month)

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [Community Platform `development` channel](https://discord.com/channels/586676777334865928/938781727017558018).

### CODE REVIEW

File: packages/components/src/ContentStatistics/ContentStatistics.tsx [hunk 1]
```diff
@@ -9,12 +11,62 @@ export interface IProps {
...
+  return (
   <Flex
     data-cy={'ContentStatistics'}
     py={1}
-    sx={{ alignItems: 'center', justifyContent: 'center', gap: 2 }}
+    sx={{ 
+      alignItems: ['flex-start','center','center'], 
+      justifyContent: 'center', 
+      gap: 2, 
+      flexDirection: ['column','row', 'row']
```
Comment (thisislawatts) [inline packages/components/src/ContentStatistics/ContentStatistics.tsx:30]: **issue(blocking):** @denyilm, it looks like there is a missing `,` here which is preventing the Preview build job from running.
https://github.com/ONEARMY/community-platform/actions/runs/6683827321/job/18160457358?pr=2952

Comment (denyilm) [reply packages/components/src/ContentStatistics/ContentStatistics.tsx]: @thisislawatts, I've added it, could we try again?

**Comment by davehakkens** [comment]
thanks @denyilm looks nice. Two questions:
1. On mobile, is it possible to make the entire "more information" section clickable and not only the dropdown icon? This would make it a bit easier and more user friendly.
2. Could we also move contributors in here on mobile? It was noted in the original issue https://github.com/ONEARMY/community-platform/issues/2735 but see now how that might have been unclear (we can also create a new issue for it if you prefer)

**Comment by denyilm** [comment]
Thanks for your feedback @davehakkens

Yes, I can make them. The second one could take a bit longer. If we want to merge it sooner without the contributors part, we can open another issue for it. Otherwise, we can keep it under the same issue and I can modify this MR.

**Comment by davehakkens** [comment]
ok. let's seperate it into another issue so this can be merged faster.
Are you able to fix the failing tests?

**Comment by denyilm** [comment]
@davehakkens Yes. 

One of them is about the commit message, I think I can fix it by changing the commit message. 

The other one fails in 'install firebase-tools globally' stage, it generates a 500 internal server error. I am not sure how related is it to the part I added, is there anyone who I can ask for help regarding that?

**Comment by davehakkens** [comment]
@thisislawatts any suggestions? 👆

**Comment by denyilm** [comment]
Hi @davehakkens! 
Since I was not able to change old commit messages on this repository, I had to open a new [PR](https://github.com/ONEARMY/community-platform/pull/2988) for this issue, it seems to have passed all the checks. Clicking on "more information" section is enabled as well.

**Comment by thisislawatts** [comment]
Thanks for the update @denyilm, I will close this PR as it is superseded by #2988

**Pull request closed without merging**